modules/osd-storage-pv-aws-create-sharedvolumes.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/
+
+[id="osd-storage-pv-aws-create-sharedvolumes-console_{context}"]
+= Creating SharedVolume resources using the console
+
+You must create one SharedVolume resource per file system:access point pair in each project from which you want pods to access it. To do this using the console:
+
+.Procedure
+
+. In the OpenShift web console, create and navigate to a project.
+. Click *Operators* -> *Installed Operators*. Find the entry for AWS EFS Operator, and click *SharedVolume* under Provided APIs.
+. Click *Create SharedVolume*.
+. Edit the sample YAML:
+.. Type a suitable value for `name`.
+.. Replace the values of `accessPointID` and `fileSystemID` with the values from the EFS resources you created earlier.
++
+[source,yaml]
+----
+  apiVersion: aws-efs.managed.openshift.io/v1alpha1
+  kind: SharedVolume
+  metadata:
+    name: sv1
+    namespace: efsop2
+  spec:
+    accessPointID: fsap-0123456789abcdef
+    fileSystemID: fs-0123cdef
+----
+
+. Click *Create*.
++
+The SharedVolume resource is created, and triggers the AWS EFS Operator to generate and associate a PersistentVolume:PersistentVolumeClaim pair with the specified EFS access point.
+
+. To verify that the PersistentVolumeClaim (PVC) exists and that it is bound, click *Storage* -> *Persistent Volume Claims*.
++
+The PVC name is `pvc-<SharedVolume_name>`. The associated PV name is `pv-<project_name>-<SharedVolume_name>`.
+
+[id="osd-storage-pv-aws-create-sharedvolumes-cli_{context}"]
```

Comment:
This will probably need to be two separate modules: one for the web console and one for the CLI. We can't have two level one sections in a single module